packages/ui/src/screens/account_details/components/staking/components/delegations/components/desktop/index.tsx

```diff
@@ -28,6 +29,7 @@ const Desktop: React.FC<{
         />
       ),
       amount: `${amount} ${x.amount?.displayDenom.toUpperCase()}`,
+      commission: `${commission} %`,
```

Comment:
```suggestion
      commission: numeral(x.commission).format('0%'),
```
suggest to format using `numeral` and remove `const commission = x.commission ? x.commission : '';`